imports/ui/components/settings/Settings.jsx

```diff
@@ -24,7 +24,12 @@ class SettingsPage extends Component {
             currencies: [],
             check1: true,
             check2: false,
-            languageSelected: 'en'
+            languageSelected: 'en',
+            name: Meteor.user().profile.fullName,
+            number: Meteor.user().profile.contactNumber || '' ,
+            username: Meteor.user().username || '',
+            email: Meteor.user().emails ? Meteor.user().emails[0].address : '',
```

Comment:
It will produce error if emails having empty array because empty array is also true. Make this `<variable>.emails && <variable>.emails.length`.